src/Mutators/__init__.py

```diff
@@ -0,0 +1,5 @@
+import glob
```

Comment:
Is this what correctly gathers the mutators so they can be imported with * later on?

Reply:
This is a Unix style pathname pattern expansion module in python, to make a path look like '/.../.../*.py'